Aspire Budgeting/Models/AspireNumber.swift

```diff
@@ -32,4 +32,11 @@ struct AspireNumber: Equatable, Hashable {
     self.decimalValue = decimalValue
     isNegative = decimalValue < 0
   }
+
+  static func /(num: AspireNumber, den: AspireNumber) -> Double {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/d10100e040432c110cd65c83489073001e61d0ab/Aspire Budgeting/Models/AspireNumber.swift#L36"></span>Operators should be surrounded by a single whitespace when defining them.<br />
<code>operator_whitespace</code> <code>AspireNumber.swift:36</code></td>
    </tr>
  </tbody>
</table>